Extensions/BBText/JsExtension.js

```diff
@@ -12,20 +13,9 @@
  * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
  */
 
-/*::
-// Import types to allow Flow to do static type checking on this file.
-// Extensions declaration are typed using Flow (like the editor), but the files
-// for the game engine are checked with TypeScript annotations.
-import { type ObjectsRenderingService, type ObjectsEditorService } from '../JsExtensionTypes.flow.js'
-*/
-
-const stringifyOptions = (options) => '["' + options.join('","') + '"]';
```

Comment:
This line is missing and yet ts doesn't complain (extension doesn't build)
How come this is happening?

Reply:
JSextensions were excluded from the tsconfig, I'm adding it back

Reply:
Sorry for my carelessness 🙏